app/views/documents/show/_history_tab.html.erb

```diff
@@ -29,10 +29,14 @@
         <% end %>
 
         <% if entry.backdated? %>
-          <% date = format_date(entry.edition.backdated_to) %>
+          <% date = format_date(entry.revision.backdated_to) %>
           <%= t "documents.history.entry_content.backdated", date: date %>
         <% end %>
 
+        <% if entry.backdate_deleted? %>
+          <%= t "documents.history.entry_content.backdate_deleted", date: date %>
```

Comment:
this date variable doesn't exist / isn't used, also I don't think we need an entry content for this action.

Reply:
Removed